src/middlewared/middlewared/plugins/vm.py

```diff
@@ -379,7 +384,7 @@ def set_tap_mtu(self, iface, tap):
         if bhyve_error == 0:
             if self.proc:
                 try:
-                    os.kill(self.proc.pid, 0)
+                    os.kill(self.proc.pid, signal.SIGHUP)
```

Comment:
Shoudnt this remain as 0? Since you just want to check if the vm is running and not "reload"?

Reply:
You are correct! I have fixed it.

Tks.